apps/sms/views/shared/test/unit/activity_handler_test.js

```diff
@@ -460,7 +460,7 @@ suite('ActivityHandler', function() {
       });
     });
 
-    suite('Close notification', function() {
+    suite.skip('Close notification', function() {
```

Comment:
Right, the object returnd by the mock for NotificationHelper is not really easy to work with.
You can replace MockNotificationHelper.send current code with:
```javascript
  send: function(titleL10n, options) {
    this.mOptions = options;
    this.mTitleL10n = titleL10n;
    var notification = {
      addEventListener: function(type, cb) {
        self.mEvents[type] = cb;
      },
      close: sinon.stub()
    };
    sinon.stub(notification, 'addEventListener');

    return Promise.resolve(notification);
  },
```

Then from your test:

```javascript
// in setup or at the start of test
this.sinon.spy(NotificationHelper, 'send');

... test stuff

NotificationHelper.send.firstCall.returnValue.then((notification) => {
  sinon.assert.called(notification.close);
}).then(done, done);
```